api/src/main/java/org/pmiops/workbench/exfiltration/EgressRemediationService.java

```diff
@@ -253,4 +278,99 @@ private void stopUserRuntimes(String userEmail) {
     int stopCount = leonardoNotebooksClient.stopAllUserRuntimesAsService(userEmail);
     log.info(String.format("stopped %d runtimes for user", stopCount));
   }
+
+  /**
+   * File a Jira investigation ticket if no open ticket exists for this VM. Identify matching
+   * tickets by the "Egress VM Prefix" and "RW Environment" custom fields on the ticket. If a ticket
+   * already exists, add a comment instead.
+   */
+  private void logEventToJira(DbEgressEvent event, EgressRemediationAction action)
+      throws ApiException {
+    String envShortName = workbenchConfigProvider.get().server.shortName;
+    SearchResults results =
+        jiraService.searchIssues(
+            // Ideally we would use Resolution = Unresolved here, but due to a misconfiguration of
+            // RW Jira, transitioning to Won't Fix / Duplicate do not currently resolve an issue.
+            String.format(
+                "\"%s\" ~ \"%s\""
+                    + " AND \"%s\" ~ \"%s\""
+                    + " AND status not in (Done, \"Won't Fix\", Duplicate)"
+                    + " ORDER BY created DESC",
+                IssueProperty.EGRESS_VM_PREFIX.key(),
+                event.getUser().getRuntimeName(),
+                IssueProperty.RW_ENVIRONMENT.key(),
+                envShortName));
+
+    if (results.getIssues().isEmpty()) {
+      jiraService.createIssue(
+          IssueType.TASK,
```

Comment:
our current egress ticket type is story.

Reply:
True. I feel task is probably better though since it should be pretty mechanical. Let me know if you disagree.